dev/travis/postfix.yaml

```diff
@@ -0,0 +1,14 @@
+# Create a merged output build/.
+before_deploy:
+  - pushd examples/hacker_news_pwa
```

Comment:
We've already built in a stage before, right? Why do it again?

...or just running it expecting a `no-opt`?

Reply:
Right, expectation is a no-op, but I'm not sure deployment can see the build cache, so being extra cautious.

Reply:
Well. It's weird.

`the -d example/hacker_new...` condition makes sure that directory exists.

So the deploy (and pre deploy) will only run in that case.

...I guess it should be a no-op, but would be good to validate.

Reply:
Definitely. Let's watch the logs and see what happens.

Reply:
Won't see anything here. Only when it lands in master.

Unless you want to push this to a branch here and enable it – to sanity check...